modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/search/SearchTransactionRequest.scala

```diff
@@ -0,0 +1,20 @@
+package org.tessellation.rosetta.server.model.search
+
+import org.tessellation.schema.address.Address
+import org.tessellation.security.hash.Hash
+
+import eu.timepit.refined.types.numeric.{NonNegLong, PosLong}
+
+/** This is an intermediary object used to hold the values from the Rosetta request that is relevant to our system. It is used by the
+  * https://www.rosetta-api.org/docs/SearchApi.html endpoints.
+  */
+
+case class SearchTransactionRequest(
+  operation: Operator,
+  address: Option[Address],
+  status: Option[OperationStatus],
+  limit: Option[PosLong],
+  offset: Option[PosLong],
+  transactionHash: Option[Hash],
+  maxBlock: Option[NonNegLong]
```

Comment:
What is the maxBlock? Which property of the block it represents?

Reply:
`maxBlock` is the `max_block` field in the request schema [here](https://www.rosetta-api.org/docs/SearchApi.html#searchtransactions). The endpoint searches for transactions inside of blocks (i.e., snapshots). `maxBlock` specifies the maximum height of the blocks to search.